lib/prometheus_exporter/middleware.rb

```diff
@@ -44,14 +44,25 @@ def call(env)
       controller = params["controller"]
     end
 
-    @client.send_json(
+    obj = {
       type: "web",
       timings: info,
       queue_time: queue_time,
       action: action,
       controller: controller,
       status: status
-    )
+    }
+    labels = custom_labels(env)
+    if labels.present? && labels.length > 0
+      obj = obj.merge(custom_labels: labels)
+    end
+
+    @client.send_json(obj)
+  end
+
+  # allows subclasses to add custom labels based on env
+  def custom_labels(env)
+    {}
```

Comment:
let's return nil here, then we can avoid all this merging in the client when labels are not configured.

Also present? is going to be an active support thing I think

Reply:
Returns `nil` and removed `present?` in the latest commit.